src/call.c

```diff
@@ -1358,6 +1357,12 @@ int call_answer(struct call *call, uint16_t scode, enum vidmode vmode)
 		return 0;
 	}
 
+	if (sipsess_awaiting_prack(call->sess)) {
+		info("call: answer: can not answer because we are awaiting a "
+		     "PRACK to a 1xx response with SDP\n");
+		return 0;
```

Comment:
Shouldn't `call_answer()` return an error code here?

Reply:
Good find, thank you. What do you think would be appropriate to return? `EPERM`? `EAGAIN`?

Reply:
Not sure. EAGAIN sounds not bad.

The other return 0 in a separate PR.

Reply:
Done.

The other fix is in #2405.